### ISSUE TITLE
TimeZoneLoader: NoSuchElementException when loading bundled time zone "EST"

### ISSUE THREAD

**Issue by rfc2822** [ical4j/ical4j#207]
When the "EST" time zone is loaded with the default ical4j time zone registry:

```
val tzRegistry = TimeZoneRegistryFactory.getInstance().createRegistry()!!
tzRegistry.getTimeZone("EST")
```

it throws a `NoSuchElementException` from within `TimeZoneLoader`:

```
java.util.NoSuchElementException
 at java.util.AbstractList$SimpleListIterator.next(AbstractList.java:59)
 at java.util.Collections.min(Collections.java:1682)
 at net.fortuna.ical4j.model.TimeZoneLoader.addTransitionRules(TimeZoneLoader.java:174)
 at net.fortuna.ical4j.model.TimeZoneLoader.generateTimezoneForId(TimeZoneLoader.java:164)
 at net.fortuna.ical4j.model.TimeZoneLoader.loadVTimeZone(TimeZoneLoader.java:109)
 at net.fortuna.ical4j.model.TimeZoneRegistryImpl.getTimeZone(TimeZoneRegistryImpl.java:181)
```

Maybe the EST definition is out of date, or there's a parsing problem in `addTransitionRules`, or both?

**Comment by benfortuna** [comment]
Fix PR #210

**Comment by benfortuna** [comment]
Fix applied. Closing issue.
